wiki/Gameplay/Accuracy/de.md

```diff
@@ -28,31 +23,48 @@ Referenz für einen Hit-Circle:
 
 ### ![](/wiki/shared/mode/taiko.png) osu!taiko
 
-![Genauigkeit = 0,5(Anzahl von GOOD + Anzahl von GREAT) / (Anzahl von BAD + Anzahl von GOOD + Anzahl von GREAT)](img/accuracy_taiko.png "Genauigkeitsformel für osu!taiko")
+![Genauigkeit = (Anzahl an GREATs + 0.5 \* Anzahl an GOODs) / (Anzahl an GREATs + Anzahl an GOODs + Anzahl an Misses)](img/accuracy_taiko.png "Genauigkeitsformel für osu!taiko")
 
 In osu!taiko wird die Genauigkeit berechnet, indem man die Summe der Notengenauigkeit (wie präzise man die Note getroffen hat) durch die Anzahl der bisher getroffenen Noten teilt. Die Notengenauigkeit wird als GREAT (良) (zählt als 100%), GOOD (可) (zählt als 50%) (halb) und MISS/BAD (不可) (zählt als 0%, was auch die Combo beendet) bezeichnet. Trommelwirbel und Spinner haben keinen Einfluss auf die Genauigkeit.
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
-![Genauigkeit = (Anzahl der Droplets + Anzahl der Drops + Anzahl der Fruits) / (Anzahl der verfehlten Droplets + Anzahl der verfehlten Drops + Anzahl der verfehlten Fruits + Anzahl der Droplets + Anzahl der Drops + Anzahl der Fruits)](img/accuracy_catch.png "Genauigkeitsformel für osu!catch")
+![Genauigkeit = (Anzahl der gefangenen Früchte + Anzahl der gefangenen Drops + Anzahl der gefangenen Droplets) / (Anzahl aller Fruits + Anzahl aller Drops + Anzahl aller Droplets)](img/accuracy_catch.png "Genauigkeitsformel für osu!catch")
 
 In osu!catch wird die Genauigkeit berechnet, indem die Gesamtzahl der gesammelten Hit-Objekte, die keine Spinner sind, durch die Gesamtzahl der Objekte, die keine Spinner sind, geteilt wird. Alle Hit-Objekte haben den gleichen Wert, mit der Ausnahme von Bananen, da diese Teil der Spinner-Objekte sind.
 
-*Hinweis für API-Benutzer: Um die Genauigkeit in osu!catch zu berechnen, befindet sich die Anzahl der Droplets unter `count50` und die Anzahl der fehlenden Droplets unter `countkatu`.*
+*Hinweise für [API](/wiki/osu!api)-Benutzer:*
+
+- Die Anzahl der Gefangenen Drops wird als `count100` zurückgegeben.
+- Die Anzahl der Gefangenen Droplets wird als `count50` zurückgegeben.
+- Die Anzahl der verfehlten Fruits *und* Drops zusammen wird als `CountMiss` zurückgegeben.
```

Comment:
```suggestion
- Die Anzahl der verfehlten Früchte *und* Drops zusammen wird als `CountMiss` zurückgegeben.
```
I noticed that you didn't replace `Fruits` here. Do you prefer to call them in their German translation `Früchte`? respektive and I negotiated to mention them as `Fruits` in more technical articles. Though, in more beginner-friendly articles, I still prefer to call them Früchte (e.g. if it was a general description of the game modes --> `Kreise klicken` instead of `(Hit-)Circles klicken`).

We can change this to `Früchte` everywhere for consistency if you want (or use `Fruits` overall). See the [hit object](https://osu.ppy.sh/wiki/de/Gameplay/Hit_object) article for details.

Reply:
I'm fine with calling them 'Fruits'. Will adjust the article.